core/state/journal.go

```diff
@@ -130,6 +130,14 @@ type (
 	touchChange struct {
 		account *common.Address
 	}
+	// Changes to the access list
+	accessListAddAccountChange struct {
+		address *common.Address
+	}
+	accessListAddSlotChange struct {
+		slot    *common.Hash
+		address *common.Address
```

Comment:
Nit, pls swap the field order, it's more meaningful because the slot is within an account, so the account has a higher sort order :P